commcare_connect/opportunity/views.py

```diff
@@ -436,7 +436,7 @@ def add_budget_existing_users(request, org_slug=None, pk=None):
         {
             "form": form,
             "opportunity_claims": opportunity_claims,
-            "budget_per_visit": opportunity.budget_per_visit,
+            "budget_per_visit": opportunity.budget_per_visit_new,
```

Comment:
For exactly this reason, we really need to rename these properties (or get rid of the old one if its not in use), but `new` isn't helpful in telling you why it's different or that you should use it.